state/aws/dynamodb/metadata.yaml

```diff
@@ -0,0 +1,83 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: state
+name: aws.dynamodb
+version: v1
+status: stable
+title: "AWS DynamoDB"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-state-stores/setup-dynamodb/
+capabilities:
+  - crud
+  - transactional
+  - etag
+  - ttl
+authenticationProfiles:
+  - title: "AccessKey and secretKey"
+    description: "Authenticate using accessKey and secretKey."
+    metadata:
+      - name: accessKey
+        type: string
+        required: false
+        sensitive: true
+        description: |
+          ID of the AWS account with appropriate permissions to SNS and SQS.
+          Can be secretKeyRef to use a secret reference.
```

Comment:
1. Is the mention of SNS/SQS correct here or a typo?
2. Please remove "Can be secretKeyRef..." since that's noit specific to this component (also below)